src/actions/commit/helpers/assertPreviousCommitValidity/index.ts

```diff
@@ -0,0 +1,34 @@
+import lint from '@commitlint/lint';
+import load from '@commitlint/load';
+import chalk from 'chalk';
+import errorAndExit from '../../../../helpers/errorAndExit';
+import { globals } from '../../../../helpers/globals';
+import promptProceedWithInvalidCommit from '../promptProceedWithInvalidCommit';
+
+const NUMBER_OF_COMMITS = 1;
+
+const assertPreviousCommitValidity = async (): Promise<void> => {
+  const { git } = globals;
+  const { all } = await git.log({ maxCount: NUMBER_OF_COMMITS });
+  const lastMessage = all[0].message;
+
+  if (!lastMessage) return;
+
+  const config = await load();
```

Comment:
What happens if a project doesn't have commitlint? I'm keen for this tool to work regardless of commitlint's existence.

Reply:
ah - didn't realise - I will have a think.  I am guessing it will just be a check to find out if it's actually been able to load some rules.  If they aren't there then skip all this

Reply:
My original code did actually cope when there wasn't a config and would handle it but I have added in some explicit checks to ensure that it's handled just in case